Themes/default/languages/ManageSettings.english.php

```diff
@@ -127,21 +127,30 @@
 $txt['cache_level2'] = 'Level 2 Caching';
 $txt['cache_level3'] = 'Level 3 Caching (Not Recommended)';
 $txt['cache_accelerator'] = 'Caching Accelerator';
-$txt['smf_cache'] = 'SMF file based caching';
+$txt['filebased_cache'] = 'SMF file based caching';
 $txt['sqlite_cache'] = 'SQLite3 database based caching';
 $txt['postgres_cache'] = 'PostgreSQL caching';
 $txt['cachedir_sqlite'] = 'SQLite3 database cache directory';
 $txt['apc_cache'] = 'APC';
 $txt['apcu_cache'] = 'APCu';
-$txt['memcache_cache'] = 'Memcache';
-$txt['memcached_cache'] = 'Memcached';
+$txt['memcacheimplementation_cache'] = 'Memcache';
+$txt['memcachedimplementation_cache'] = 'Memcached';
```

Comment:
is this an error?

Reply:
Nope, memcache and memcached are now separated, thus they have their own txt strings even if the only thing that changes is adding a "d".

The "Implementation" part is just being safe as to not be confused with the real Memcache and Memcached classes since we can't really fully use namespaces.